### PR TITLE
Add the stack auditor, a plugin that helps operators know which stacks are in use on the platform, to the cli plugin

### PR DESCRIPTION
**Pull request by Benjamintf1**
# Submitting Plugins

If you haven't yet, please review our contributing guidelines:  
https://github.com/cloudfoundry/cli-plugin-repo#submitting-plugins

In particular ensure the following requirements are being met:
* [x] The plugin's `name` field in `repo-index.yml` matches the `Name` field in the plugin's `plugin.PluginMetadata` section
* [x] The plugin's `url` field in `repo-index.yml` contains the same version from the `version` field

## Description of the Change

Add the stack auditor, a plugin that helps operators know which stacks are in use on the platform, to the cli plugin registry

### CODE REVIEW

File: repo-index.yml [hunk 1]
```diff
@@ -1676,6 +1676,28 @@ plugins:
   name: spring-cloud-dataflow-for-pcf
   updated: "2020-03-04T09:47:00Z"
   version: 1.0.2
+- authors:
+  - name: stack-auditor team
```
[the remark this answers is not in the document]
Comment (Benjamintf1) [reply repo-index.yml:1680]: Homepage is already included later on

File: repo-index.yml [hunk 2]
```diff
@@ -1676,6 +1676,28 @@ plugins:
   name: spring-cloud-dataflow-for-pcf
   updated: "2020-03-04T09:47:00Z"
   version: 1.0.2
+- authors:
```
Comment (Samze) [inline repo-index.yml:1679]: **suggestion** Think you may need to run `go run sort/main.go repo-index.yml`  else a test will fail

Comment (Benjamintf1) [reply repo-index.yml]: 😔 I thought I had put it in the right place. Fixed.

**Review by Samze — APPROVED** [review]

**Comment by gururajsh** [comment]
LGTM

**Pull request merged**